src/main/java/stringformula/Number.java

```diff
@@ -0,0 +1,47 @@
+package stringformula;
+
+import stringformula.exception.InvalidNumberValueException;
+
+import java.util.Objects;
+
+public class Number {
+    private final int value;
+
+    public Number(int value) {
+        if (value < 0) {
```

Comment:
`0`에 의미를 부여하여 가독성을 높여 보면 어떨까요?
```suggestion
        if (value < MINIMUM) {
```